cloudstate-kotlin-support/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+plugins {
+    kotlin("jvm") version "1.3.72"
+    id("com.lightbend.akka.grpc.gradle") version "0.8.4"
+    `maven-publish`
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(kotlin("stdlib-jdk8"))
+    implementation(kotlin("reflect"))
+    api("io.cloudstate:cloudstate-java-support:0.4.3")
+    implementation("net.bytebuddy:byte-buddy:1.10.7")
+    implementation("net.bytebuddy:byte-buddy-agent:1.10.7")
+    implementation("org.slf4j:slf4j-api:1.7.25")
+    implementation("ch.qos.logback:logback-classic:1.2.3")
```

Comment:
I'm not sure, that logback should be here. Users should self choose a logger.

Reply:
Good point!